config/routes.rb

```diff
@@ -156,7 +156,7 @@ def patch(*)
         get "tl3_requirements"
         put "anonymize"
         post "merge"
-        post "reset_bounce_score"
+        post "reset-bounce-score" => "users#reset_bounce_score"
```

Comment:
I think Rails is smart enough to find the correct name for the action.

```suggestion
        post "reset-bounce-score"
```